cmd/security-secrets-setup/README.md

```diff
@@ -1,16 +1,88 @@
 # Testing Instruction for security-secrets-setup
 
-## Build
+## Executable Build
 
-Use Makefile in the base directory of this repository to build the docker image of security-secretes-setup:
+Use Makefile in the base directory of this repository to build the binary executable of security-secrets-setup:
+
+```sh
+make cmd/security-secrets-setup/security-secrets-setup
+```
+
+It should create a binary executable under the directory *cmd/security-secrets-setup/* if successfully built.
+
+## Executable Run [generate] subcommand
+
+To run the **generate** subcommand in linux system, change directory to *cmd/security-secrets-setup/* and then run 
+
+```sh
+sudo -E ./security-secrets-setup generate
+```
+
+Verify that it runs successfully with generating both Vault and Kong's TLS assets under `/run/edgex/secrets/` directory.
+
+Clean up `/run/edgex/secrets` once done verification if necessary.
+
+## Executable Run [cache] subcommand
+
+Before cache subcommand is issued, make sure cache dir (in this example, it is */tmp/pki/cache*) is non-existing or empty.
+
+To run the **cache** subcommand in linux system, change directory to *cmd/security-secrets-setup/* and then run 
+
+```sh
+mkdir -p /tmp/pki/cache && export PKI_CACHE=/tmp/pki/cache && sudo -E ./security-secrets-setup cache
+```
+
+After successfully run, verify that there are Vault and Kong TLS assets generated and cached into cache dir */tmp/pki/cache* and also deployed from cache directory into the deployed directory */run/edgex/secrets* via `ls` command.
+
+One can also use some file diff tool to compare the TLS files like:
+
+```sh
+sudo diff -r /tmp/pki/cache/ca/ca.pem /run/edgex/secrets/ca/ca.pem
+```
+
+If compares with the whole parent directories between cache dir and deployed dir, one should only find the difference in generated **sentinel** files.
+
+```sh
+sudo diff -r /tmp/pki/cache/ /run/edgex/secrets/
+
+Only in /run/edgex/secrets/ca: .security-secrets-setup.complete
+Only in /run/edgex/secrets/edgex-kong: .security-secrets-setup.complete
+Only in /run/edgex/secrets/edgex-vault: .security-secrets-setup.complete
+```
+
+Clean up `/run/edgex/secrets` once done verification if necessary.
+
+## Executable Run [import] subcommand
+
+To run the **import** subcommand in linux system, change directory to *cmd/security-secrets-setup/* and then run 
+
+```sh
+mkdir -p /tmp/pki/cache && export PKI_CACHE=/tmp/pki/cache && sudo -E ./security-secrets-setup import
+```
+
+There are two cases can happen depends on the status of cache directory:
+
+1. Cache dir */tmp/pki/cache* is empty:
```

Comment:
Since there's only 2 possibilities, I think it's simpler to just use plain English and not enumerate them in a number list, like so:

```suggestion
If the cache dir `/tmp/pki/cache` is empty, then the command blocks and waits for ....
```

(also note this now out-of-date per our Slack conversation regarding the blocking...)

Reply:
i think list reads better, I agree with you on not using the number list but I am ok with changing it to a bullet list not numbered since number list implies the sequence and priority